test/libsolidity/semanticTests/uninitializedFunctionPointer/invalidInConstructor.sol

```diff
@@ -0,0 +1,23 @@
+contract C {
+
+  function() internal storedFn;
+
+  bool flag;
+
+  constructor() public {
+    if (!flag) {
```

Comment:
Why the flag in the tests?

Reply:
Ah ok, I think I got it now

Reply:
If the invalid jump tag is zero, it will jump to the start of the constructor again where the flag is set to true, but it actually is not like that - the failure case is just another tag that might or might not exist.